build.gradle

```diff
@@ -46,8 +43,8 @@ allprojects {
     version = '2.5-SNAPSHOT'
```

Comment:
This needs updating to 2.7-SNAPSHOT or I can integrate the PR I previously created at https://github.com/odpi/egeria-database-connectors/pull/13 - but given it has other changes that yours superceed I think you updating is probably the best